CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+## Guidelines
+
+- For commit messages, make sure to follow the discussion [here](https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53) as much as possible.
+- Comment or let yourself be assigned to an issue before working on it. **Do not** work on an issue that someone has already commented or been assigned on, unless it is known that they are not working on it anymore.
+- Create a separate branch for the feature/issue you are working on. For example, if you are working on the landing page, you run this command on your terminal or command prompt before starting your work:
+
+```bash
+git checkout -b ui/landing-page
```

Comment:
Maybe make this a bit more generic `git checkout -b feat/<REPLACE_ME>`?
I wonder if someone would get confused here and always use this exact branch name.

Reply:
Fair enough